chainer/functions/evaluation/classification_summary.py

```diff
@@ -90,6 +90,9 @@ def classification_summary(y, t, label_num=None, beta=1.0, ignore_label=-1):
     into account for calculating the above quantities.
     By default, it is set to -1 so that all instances are taken
     into consideration, as labels are supposed to be non-negative integers.
+    If ``ignore_label`` is "valid", that is, it is a non-negative
+    integer less than ``label_num``, then the ``ignore_label``-th
+    entries in output arrays do not contain correct quantities.
```

Comment:
I have several issues which I do not have clear answers.
* By raising `RuntimeWarning`, we declare that setting a non-negative integer to `ignore_label` is an abuse of this function. But I guess some users sometimes would like to use `ignore_label` in this way to ignore some specified label. So to me we should not raise warning here.
* We will be in the position that we should invalidate to assign an non-negative integer to `ignore_label` in this paragraph. The message attached to this `RuntimeWarning` is division-by-zero warning. It is by product of the inappropriate `ignore_label` and user cannot understand the cause of the warning easily. So we should wrap this warning and attach the more meaningful message.

Thank you for your suggestion on adding description on ``